src/plots/barPlot.ts

```diff
@@ -298,6 +299,24 @@ export module Plots {
       return Plottable.Utils.DOM.intersectsBBox(xRange, yRange, barBBox);
     }
 
+    protected _datumVisibleOnPlot(pixelPoint: Point, datum: any, index: number, dataset: Dataset) {
```

Comment:
The new ordering makes more sense because the `datum`, `index`, `dataset` are required for the accessor, so they belong to the same logical unit. I am not sure where else we have `datum`, `pixelPoint`, `index`, `dataset` in this order?